test/multifile/typealias/two-modules/library.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: true
```

Comment:
I don't consider cross-module issues to *be* multi-file support. The code paths are almost completely different.

Maybe this is just an issue of terminology, but I don't think cross-module tests should be mixed in with intra-module tests, regardless of the directory name.